tests/functional/audit/test_api.py

```diff
@@ -0,0 +1,190 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test API contract via client"""
+
+from enum import Enum
+from itertools import chain
+from pathlib import Path
+from typing import Collection, Union, Tuple, Callable
+
+import allure
+import pytest
+import requests
+from adcm_client.audit import ObjectType, OperationResult, OperationType, LoginResult
+from adcm_client.objects import ADCMClient, Group, User
+from coreapi.exceptions import ErrorMessage
+
+from tests.functional.rbac.conftest import BusinessRoles
+from tests.library.assertions import sets_are_equal
+
+# pylint: disable=redefined-outer-name
+
+BUNDLES_DIR = Path(__file__).parent / 'bundles'
+
+
+def _check_audit_logs(
+    endpoint: str, operation: Callable, token: str, log_field_name: str, filters: Union[Enum, Collection[str]]
+):
+    for filter_value in filters:
+        simple_type_filter = filter_value.value if isinstance(filter_value, Enum) else filter_value
+        with allure.step(f'Check filter {log_field_name}={simple_type_filter}'):
+            plain_argument = {log_field_name: filter_value}
+            string_filter_args = {log_field_name: simple_type_filter}
+            logs_from_client = operation(**plain_argument)
+            # prepare all type of objects (?)
```

Comment:
what does this comment mean?

Reply:
It was about preparing operations (create/update/delete) for each object type, it's redundant now, I'll remove it